controllers/mail/MessageController.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace humhub\modules\smartVillage\controllers\mail;
+
+use humhub\modules\rest\components\BaseController;
+use yii\db\Query;
+use Yii;
+use yii\data\Pagination;
+
+class MessageController extends BaseController
+{
+    /**
+     * Fetching conversation's data from both message and user_message of a user
+     * @return array
+     */
+     public function actionIndex(){
+         $results = [];
+
+         $messagesQuery = new Query();
+         $messagesQuery->select("*")->from('message')
+                        ->leftJoin('user_message', 'user_message.message_id = message.id')
+                        ->where(['user_id' => Yii::$app->user->id]);
+
+         foreach ($messagesQuery->all() as $message) {
+             $results[] = self::getMessage($message);
```

Comment:
is everything "the same" as for `/mail` but this `getMessage` with our `status` and `seen_at`?

Reply:
Hi @donni106 
Yes all keys are same. we are added two more keys 'status' and 'seen_at'

Reply:
i did not meant the additional keys but all the other code in this controller? is it the same as for /mail from HumHub?

Reply:
Hi @donni106 
Yes, the code is same as /mail. We pushed another commit now. In this commit we refactor the code. please review it once
Thanks!

Reply:
nice, thanks